docs/en/getting-started/index.md

```diff
@@ -11,7 +11,7 @@ pagination_next: 'en/tutorial'
 We have a lot of resources for helping you get started and learn how ClickHouse works:
 
 - If you need to get ClickHouse up and running, check out our [Quick Start](../quick-start.mdx)
-- The [ClickHouse Tutorial](../tutorial.md) analyzes a dataset of New York City taxi rides
+- The [ClickHouse Tutorial](./example-datasets/nyc-taxi.md) analyzes a dataset of New York City taxi rides
```

Comment:
`https://clickhouse.com/docs/en/tutorial` does exactly the things that are described here.

The link to the `nyc-taxi.md` is in line 22 and the description accurately reflects the content.
```
- The [New York Taxi Data](../getting-started/example-datasets/nyc-taxi.md) has an example of how to insert data from S3 into ClickHouse
```